doc/source/user_guide/indexing.rst

```diff
@@ -920,6 +920,10 @@ and :ref:`Advanced Indexing <advanced>` you may select along more than one axis
 
    df2.loc[criterion & (df2['b'] == 'x'), 'b':'c']
 
+Note- pandas does not allow indexing with NA values. Attempting to do so
```

Comment:
use a proper note construct (search for note and you'll see how).

add a ref to the correct location